scripts/mixins/thaumcraft.zs

```diff
@@ -0,0 +1,169 @@
+#loader mixin
```

Comment:
I see that file `bansoukou/Thaumcraft-1.12.2-6.1.BETA26/thaumcraft/common/lib/crafting/ThaumcraftCraftingManager.class` was removed, but I cant see any lines in `thaumcraft.zs` that related to this removal.

Tweaks in `ThaumcraftCraftingManager.class` necessary to complitely remove generation of aspects from crafting table recipes. On my machine this saving up to 40 seconds of load time.
Actually, this problem is resolved by ThaumicSpeedup mod, and this why this option is disabled in config by default.
But, in dev environment i required to rapidly changing mod count, what caused aspects to recalculate again. Thus, i need ability to disable this automation on will.

Reply:
I need to profile game load to test if changes applied.

Is there an other way to test if changes appied? This what `.mixin.out` shows me:
```java
        }
        AspectList ret = ThaumcraftCraftingManager.generateTagsFromRecipes(stack, history);
        ret = ThaumcraftCraftingManager.capAspects(ret, 500);
        ThaumcraftApi.registerObjectTag(is, ret);
```

```java
    }

    private static AspectList generateTagsFromRecipes(ItemStack stack, ArrayList<String> history) {
        AspectList ret = null;
        boolean value = false;
        ret = ThaumcraftCraftingManager.generateTagsFromCrucibleRecipes(stack, history);
        if (ret != null) {
            return ret;
        }
        ret = ThaumcraftCraftingManager.generateTagsFromInfusionRecipes(stack, history);
        if (ret != null) {
            return ret;
        }
        ret = ThaumcraftCraftingManager.generateTagsFromCraftingRecipes(stack, history);
        return ret;
    }

```